lib/tests/function.rs

```diff
@@ -2947,6 +2947,23 @@ async fn function_string_concat() -> Result<(), Error> {
 	Ok(())
 }
 
+#[tokio::test]
+async fn function_string_contains() -> Result<(), Error> {
+	let sql = r#"
+		RETURN string::contains("abcdefg", "bcd");
+	"#;
```

Comment:
Could we just add a couple of other examples/tests here:
1. `string::contains("abcdefg", "");`
2. `string::contains("abcdefg", "hij");`

Reply:
I added some more, maybe I overdid it, but I don't think that is possible for tests 👍🏼 
Let me know if that is better